include/predict/predict.h.in

```diff
@@ -287,6 +287,8 @@ struct predict_observation {
 	double range_x, range_y, range_z; 
 	///Range velocity (km/s) 
 	double range_rate;      
+	///Visibility status, whether satellite can be seen through optical means
```

Comment:
My understanding: The satellite is "visible" if all of these apply:

* The satellite is in sunlight
* The satellite is above the horizon
* The sun is low enough to make the sky so dark that the satellite is visible (this obviously depends on a definition of darkness)

The short description of the variable is OK, but I'd like to see a longer description containing all the details.